### DISCUSSION TITLE
PC Steam version

### DISCUSSION THREAD

**Discussion by TokuhasDoku** [thatskymod/Sky-CotL-Scripts#86] [Q&A] [answered]
are we able to get a pc version of all these features, i only have an Iphone and cannot get any of these to work on it and would much rather be on pc anyway

**Comment by thatskymod** [comment, marked as answer]
Hello,

I believe we can consider this case resolved given that we now have a stable, well-maintained customization for the PC/Steam version. With the new [installer app](https://github.com/XeTrinityz/ThatSkyApp/releases), downloading and deploying the mod should be straightforward.

Credits to @XeTrinityz for making this possible.

Thank you for your contribution!
